src/Bicep.Core/CodeAction/Fixes/CodeFixHelper.cs

```diff
@@ -56,4 +53,63 @@ public static CodeFix GetCodeFixForMissingBicepParams(ProgramSyntax program, Imm
             CodeFixKind.QuickFix,
             new(program.EndOfFile.Span, additionalProgram.ToString()));
     }
+
+    public static CodeFix GetCodeFixForMissingBicepExtensionConfigAssignments(ProgramSyntax program, IReadOnlyList<(string Alias, ObjectLikeType ExpectedConfigType)> missingExtAliasesWithExpectedType)
+    {
+        var terminatingNewlines = program.Children.LastOrDefault() is Token { Type: TokenType.NewLine } newLineToken ? StringUtils.CountNewlines(newLineToken.Text) : 0;
+
+        var newSyntaxList = new List<SyntaxBase>();
+
+        var precedingNewlineSyntax = terminatingNewlines switch
```

Comment:
Rather than appending the file, would it make more sense to insert the extension config assignment syntax nodes after the using statement and before the first parameter assignment?